frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -771,6 +771,23 @@ const plugin: Plugin<ConsumedExtensions> = [
         ).then((m) => m.default),
     },
   },
+  {
+    type: 'ClusterServiceVersion/Action',
+    properties: {
+      kind: 'CephBlockPool',
+      label: '%ceph-storage-plugin~Edit Block Pool%',
+      apiGroup: models.CephBlockPoolModel.apiGroup,
+      override: 'Edit',
+      callback: (kind, blockPoolConfig) => () => {
+        const blockPoolObject = { blockPoolConfig };
+        import('./components/modals/block-pool-modal/update-block-pool-modal')
+          .then((m) => m.updateBlockPoolModal(blockPoolObject))
+          .catch((e) => {
+            throw e;
+          });
+      },
+    },
+  },
```

Comment:
Add this with the extension PR. Currently its causing test failure
```suggestion
```

Reply:
if i add this due to import('./components/modals/block-pool-modal/update-block-pool-modal'), extension PR test will fail, as per hierarchy extension PR has to go first, so i can make test fail there

Reply:
Okay, you can send another PR for enabling the edit action then.